src/components/ServiceDetails/AboutService/ServiceDemo/index.js

```diff
@@ -77,21 +79,18 @@ class ServiceDemo extends Component {
     const {
       service: { org_id: orgId },
       groupInfo: { group_id: groupId },
-      wallet,
       fetchWallet,
       startFetchWalletLoader,
       stopLoader,
+      firstTimeFetchWallet,
+      updateFirstTimeFetchWallet,
     } = this.props;
-    if (isEmpty(wallet)) {
+    if (firstTimeFetchWallet) {
```

Comment:
Do you want to start polling after the API is complete ?

Reply:
Yh. we need to poll to get the latest channel balance.